src/app/vault/view.component.html

```diff
@@ -276,4 +276,7 @@
     <button appBlurClick class="primary" (click)="edit()" appA11yTitle="{{'edit' | i18n}}">
         <i class="fa fa-pencil fa-fw fa-lg" aria-hidden="true"></i>
     </button>
+    <button appBlurClick class="primary" *ngIf="!cipher?.organizationId" (click)="clone()" appA11yTitle="{{'clone' | i18n}}">
```

Comment:
Is this new syntax? Can you do `?.` now?

Reply:
[https://angular.io/guide/template-syntax#the-safe-navigation-operator----and-null-property-paths](url)

The Elvis operator can be used in templating!